h/services/user_signup.py

```diff
@@ -98,8 +107,9 @@ def signup(self, require_activation=True, **kwargs):
         # FIXME: this is horrible, but is needed until the
         # notification/subscription system is made opt-out rather than opt-in
         # (at least from the perspective of the database).
-        sub = Subscriptions(uri=user.userid, type="reply", active=True)
-        self.session.add(sub)
+        self.subscription_service.set_subscriptions(
```

Comment:
We could iterate over Type.ALL here instead of having the multiple methods in the service